test/Microsoft.AspNetCore.Mvc.FunctionalTests/ApplicationModelTest.cs

```diff
@@ -126,5 +126,25 @@ public ApplicationModelTest(MvcTestFixture<ApplicationModelWebSite.Startup> fixt
             var body = await response.Content.ReadAsStringAsync();
             Assert.Equal("From Header - HelloWorld", body);
         }
+
+        [Fact]
+        public async Task ActionModelSuppressedForPathMatching_CannotBeRouted()
+        {
+            // Arrange & Act
+            var response = await Client.GetAsync("Home/CannotBeRouted");
+
+            // Assert
+            Assert.Equal(HttpStatusCode.NotFound, response.StatusCode);
+        }
+
+        [Fact]
+        public async Task ActionModelNotSuppressedForPathMatching_CanBeRouted()
```

Comment:
I was of verifying suppressing one attribute route doesn't affect the other. I can reword the test name